cdap-ui/app/cdap/components/CaskWizards/PublishPipeline/index.js

```diff
@@ -105,7 +117,27 @@ export default class PublishPipelineWizard extends Component {
     };
     let currentNamespace = NamespaceStore.getState().selectedNamespace;
     let draftId;
-    if (this.props.input.action.type === 'create_pipeline_draft') {
+    // this is the case when this is used in an usecase, so
+    // PublishPipelineUsecase passes the CTA info to this component
+    if (this.props.buildSuccessInfo) {
```

Comment:
So the existing condition is this https://github.com/caskdata/cdap/blob/release/4.1/cdap-ui/app/cdap/components/CaskWizards/PublishPipeline/index.js#L108. We might want to keep that and have this in addition to that.